src/services/item/plugins/action/test/ws.test.ts

```diff
@@ -1,20 +1,22 @@
 import { StatusCodes } from 'http-status-codes';
-import waitForExpect from 'wait-for-expect';
+import waitForExpectDefault from 'wait-for-expect';
 
 import { FastifyInstance } from 'fastify';
 
 import { HttpMethod } from '@graasp/sdk';
 
-import { clearDatabase } from '../../../../../../test/app';
-import { TestWsClient } from '../../../../websockets/test/test-websocket-client';
-import { setupWsApp } from '../../../../websockets/test/ws-app';
-import { ItemTestUtils } from '../../../test/fixtures/items';
+import { clearDatabase } from '../../../../../../test/app.js';
+import { TestWsClient } from '../../../../websockets/test/test-websocket-client.js';
+import { setupWsApp } from '../../../../websockets/test/ws-app.js';
+import { ItemTestUtils } from '../../../test/fixtures/items.js';
 import {
   ItemOpFeedbackErrorEvent,
   ItemOpFeedbackEvent,
   memberItemsTopic,
-} from '../../../ws/events';
-import { ActionRequestExportRepository } from '../requestExport/repository';
+} from '../../../ws/events.js';
+import { ActionRequestExportRepository } from '../requestExport/repository.js';
+
+const waitForExpect = waitForExpectDefault.default;
```

Comment:
Would having this in a util easier? And you export it and use this proxy file instead of setting the default each time?

Reply:
Actually I just copied the `wait-for-export` package code in a utils file. It works better and we do not need to worry about those import issues.